en/docs/install-and-setup/setup/advance-configurations/configuring-caching.md

```diff
@@ -150,6 +152,19 @@ The OAuth token is saved in this cache, which is enabled by default. Whenever a
 
 Please refer [Response Caching]({{base_path}}/learn/api-gateway/response-caching/) to see how to enable response caching for a given API.
 
+### JWT claims cache
+
+Please refer [JWT claims Caching]({{base_path}}/learn/api-gateway/passing-end-user-attributes-to-the-backend/passing-enduser-attributes-to-the-backend-using-jwt/#expiry-time-of-the-jwt) to see how to enable JWT claims caching.
+
+### Publisher cache
+
+-   **Publisher-roles cache:** This indicates whether the role cache need to be enabled in the publisher. If this is disabled, there will be a call to key manager to all the calls to API publisher APIs. It expires in 15 minutes by default. It is highly recommended to enable this cache. However, if the system is in a state, where the role addition and deletion happens seamlessly, the cache will be in invalid state.
```

Comment:
Is this enabled by default? Please add that also in the description.

Reply:
Also what is meant by 'the cache will be in invalid state'

Reply:
Rephrased https://github.com/wso2/docs-apim/pull/1837/commits/7d1a9552c8ba33ca5ee58c512d58463aa75330ce